### DISCUSSION TITLE
Tasks

### DISCUSSION THREAD

**Discussion by ThijsRuigrok** [FieldRobotEvent/Virtual_Field_Robot_Event#2] [General]
When will the description of the tasks come available?

**Comment by FieldRobotEvent** [comment]
Hi Thijs, 
Thank you for your question.
The new tasks will be presented at our [website](https://www.fieldrobot.com/event/) as soon as possible. Until then you can have a look at the tasks of 2019. The new task will be more or less similar.

**Reply by ThijsRuigrok** [reply]
Thank you for the swift reply. I will regularly check the website for updates.  I’m looking forward to the event! :)
